### PR TITLE
NO-JIRA: setup preview site

### PR DESCRIPTION
**Pull request by stfurkan**
# READY FOR REVIEW

# Summary
- setup preview site

# Review By (Date)
- End of sprint

# Criticality
- 5

# Review Tasks

## Setup tasks and/or behavior to test

1. Check out this branch
2. Review code
3. `merge-to-preview.yml` should merge `dev` branch to `preview` branch
4. `/preview` route should only available if env value `NEXT_SITE_PREVIEW="true"`
5. `/preview` route should render page using SSR

### CODE REVIEW

**Review by sherakama — APPROVED** [review]
Works for me. Ty.

File: next.config.js [hunk 1]
```diff
@@ -39,6 +39,7 @@ module.exports = {
         ADAPT_AUTH_SAML_ENTITY: '',
         SECRET: '',
         SITE_URL: appUrl,
+        NEXT_SITE_PREVIEW: '',
```
Comment (sherakama) [inline next.config.js:42]: Throwing this in vault or Netlify Env Vars?

Comment (stfurkan) [reply next.config.js:42]: I deleted everything in Netlify Env Vars for this repo. We are only using Vault.

**Comment by stfurkan** [comment]
> Works for me. Ty.

Thanks for the review, @sherakama :)

**Pull request merged**